### ISSUE TITLE
Set values to missing

### ISSUE THREAD

**Issue by ymer** [TidierOrg/TidierData.jl#38]
In standard Julia, I can set a column to missing in this way:
`df.year .= missing`

In Tidyverse I can set a column to missing like this:
`mutate(df, year = NA)`

The Tidier equivalent would be:
`@mutate(df, year = missing)`

However, this does not work.

**Comment by kdpsingh** [comment]
Thanks for catching this issue. This is totally fixable.

The issue is that TidierData.jl thinks you are referring to a column named `missing` instead of the value of `missing`.

For now, you can use `!!` to indicate that you're referring to the value rather than the column name.

This *should* work, but I haven't tested it.

```
@mutate(df, year = !!missing)
```

However, for common keywords like `missing`, `true`, and `false`, we will fix this in TidierData.jl so that this workaround isn't needed.

**Comment by ymer** [comment]
I have tried
`@mutate(df, year = !!missing)`
and
`@mutate(df, year = [!!missing])`

Neither work

**Comment by kdpsingh** [comment]
Thanks. We will look and get this fixed.

**Comment by kdpsingh** [comment]
This is now fixed in #69. Both of these should now work:

```julia
julia> df = DataFrame(a = 1:5)
5×1 DataFrame
 Row │ a     
     │ Int64 
─────┼───────
   1 │     1
   2 │     2
   3 │     3
   4 │     4
   5 │     5

julia> @chain df @mutate(b = missing, c = !!missing)
5×3 DataFrame
 Row │ a      b        c       
     │ Int64  Missing  Missing 
─────┼─────────────────────────
   1 │     1  missing  missing 
   2 │     2  missing  missing 
   3 │     3  missing  missing 
   4 │     4  missing  missing 
   5 │     5  missing  missing 
```
